docs/content/modeling/migrating-relations.mdx

```diff
@@ -0,0 +1,310 @@
+---
+sidebar_position: 10
+slug: /modeling/migrating-relations
+---
+
+import {
+  AuthzModelSnippetViewer,
+  CardBox,
+  CheckRequestViewer,
+  DocumentationNotice,
+  ProductConcept,
+  ProductName,
+  ProductNameFormat,
+  ReadRequestViewer,
+  RelatedSection,
+  RelationshipTuplesViewer,
+  WriteRequestViewer,
+} from '@components/Docs';
+
+# Migrating Relations
+
+<DocumentationNotice />
+
+Over the course of a software lifetime, we will need to make changes to an <ProductConcept section="what-is-an-authorization-model" linkName="authorization model" /> and our goal is to avoid service disruption. In this guide we are going to discuss the steps needed when making changes to existing models.
+
+## Before You Start
+
+In order to understand this guide correctly you must be familiar with some <ProductConcept /> and know how to develop the things that we will list below.
+
+<details>
+<summary>
+
+You will start with the _<ProductConcept section="what-is-an-authorization-model" linkName="authorization model" />_ below,
+it represents a `document` _<ProductConcept section="what-is-a-type" linkName="type" />_ that can have users assigned as editor relation. Editor can edit document.
```

Comment:
```suggestion
## Prerequisites 

This guide assumes you are familiar with the following OpenFGA concepts:

- A <ProductConcept section="what-is-a-type" linkName="Type" />: a class of objects that have similar characteristics
- A <ProductConcept section="what-is-a-user" linkName="User" />: an entity in the system that can be related to an object
- A <ProductConcept section="what-is-a-relation" linkName="Relation" />: is a string defined in the type definition of an authorization model that defines the possibility of a relationship between an object of the same type as the type definition and a user in the system
- An <ProductConcept section="what-is-an-object" linkName="Object" />: represents an entity in the system. Users' relationships to it can be define through relationship tuples and the authorization model
- A <ProductConcept section="what-is-a-relationship-tuple" linkName="Relationship Tuple" />: a grouping consisting of a user, a relation and an object stored in <ProductName format={ProductNameFormat.ShortForm} />
- [Intersection Operator](../configuration-language.mdx#the-intersection-operator): the intersection operator can be used to indicate a relationship exists if the user is in all the sets of users

## Review the existing model

The document below is an example of a relational authorization model. In this model, you can assign users to the `editor` relation. The `editor` relation has write privileges that regular users do not.

In this scenario, you will migrate the following model:
```